src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothServerSocket.cs

```diff
@@ -132,6 +132,10 @@ private static void UnregisterAcceptStateChangedEvent()
         internal BluetoothServerSocket(int socketFd)
         {
             Log.Info (Globals.LogTag, "Constructing server socket");
+
```

Comment:
How about call default constructor ?
``` c#
        internal BluetoothServerSocket(int socketFd) : this()
        {
            this.socketFd = socketFd;
        }
```

or remove unused default constructor

and Constructor should be placed the top of class definition
(next of field declare)

Reply:
I think,, this concept was already discussed in the previous pull request - https://github.com/Samsung/TizenFX/pull/598
* My patchset is just fixing the bug in C# BT dll.

@ Wootak
- Is it possible to answer above question?

Reply:
I removed unused default constructor and move this constructor to top of class definition.